lib/kredits.js

```diff
@@ -81,6 +81,10 @@ class Kredits {
     return this.contractFor('Proposal');
   }
 
+  get Operator() {
+    return this.Proposal();
```

Comment:
You don't need the `()` see `get Contributors()`

Reply:
ah ja... thx.